Maestro/subscriptions.json

```diff
@@ -765,6 +765,7 @@
         "https://github.com/dotnet/project-system/blob/master/**/*",
         "https://github.com/dotnet/project-system/blob/release/**/*",
         "https://github.com/dotnet/project-system-tools/blob/master/**/*",
+        "https://github.com/dotnet/runtime/blob/darc-master/**/*",
```

Comment:
Is this going to see a lot of churn that's irrelevant to most people `fetch`ing dotnet/runtime, like the `darc-master-*` branches now? 😕

Reply:
Well, I guess this PR is simply syncing those branches to internal. It's sad that they're kept on the main fork but that's a parallel issue. (https://github.com/dotnet/arcade/issues/1211)

Reply:
It is interesting that this means the internal repo *will* increase in refs/size over time, not just be noisy, since Darc/Maestro++ won't know about the mirrored branches to delete. Unless there's some special handling.

Reply:
There's not :/ This is so runtime can easily run official builds when reacting to changes.